### DISCUSSION TITLE
密钥清单不全，跳过解锁。密钥缺失，跳过解锁。

### DISCUSSION THREAD

**Discussion by vipxuhao** [pjy612/SteamManifestCache#904] [Question / Answer]
By pjy612 v1.0.9.1 本软件永久免费，禁止用于商业或非法用途
当前Steam路径：e:/program files (x86)/steam
Steam 路径可在配置文件中自行指定
获取请求阈值...
Free到期时间:2024/11/15 0:00:00
当前Github可用请求数:4922
Github重置时间:2024/11/4 3:35:35
仓库或配置更新时间：2024/11/4 0:34:05
App:1622780 方案二 尝试获取清单列表...
App:1622780 方案二 获取到清单列表...
下载完毕:1622781_5119808912995598619.manifest
下载完毕:1622782_5909183315517509124.manifest
下载完毕:1622783_4318044060538644335.manifest
App 1622780 游戏配置加载完毕，开始进行入库配置...
App 1622780 已导入清单
App 1622780 1622781 密钥清单不全，跳过解锁。
App 1622780 1622782 密钥清单不全，跳过解锁。
App 1622780 1622783 密钥清单不全，跳过解锁。
App 1622780  密钥缺失，跳过解锁。
入库失败，请自行确认密钥状态 和 正确安装解锁工具(如 SteamTools 或 GreenLuma

这种情况是有清单无密钥吗

**Comment by pjy612** [comment]
是的 密钥自己想办法
